test/ssl/test.sh

```diff
@@ -237,7 +275,7 @@ test_client_ssl() {
 	echo "host all all 127.0.0.1/32 trust" > pgdata/pg_hba.conf
 	echo "host all all ::1/128 trust" >> pgdata/pg_hba.conf
 	reconf_pgsql "ssl=on" "ssl_ca_file='root.crt'"
-	psql_bouncer -q -d "dbname=p0 sslmode=require" -c "select 'client-ssl-connect'" | tee tmp/test.tmp
+	psql_bouncer -q "dbname=p0 sslmode=require" -c "select 'client-ssl-connect'" | tee tmp/test.tmp
```

Comment:
This change doesn't make sense.

Reply:
I think I changed this when the tests were not working correctly. It didn't help, because there was an actual bug, but then I didn't change it back.